tests/health_test.go

```diff
@@ -0,0 +1,35 @@
+package test
+
+import (
+	"net/http"
+	"testing"
+
+	. "github.com/smartystreets/goconvey/convey"
+)
+
+// This tests the /health endpoint
+func TestProject(t *testing.T) {
+	Convey("If API exists", t, func() {
+		baseUrl := getBaseUrl(ENV)
+		So(baseUrl, ShouldNotEqual, "")
+
+		Convey("When consuming the /health endpoint", func() {
+			res, body, err := Health(baseUrl)
+
+			Convey("Then it should return a 200 with the correct body", func() {
+				So(err, ShouldBeNil)
+				So(res, ShouldNotBeNil)
+				So(body, ShouldNotBeNil)
+				So(res.StatusCode, ShouldEqual, http.StatusOK)
+				for k, v := range body {
```

Comment:
This is useful when you have to check many stuff but you don't need to worry about the details. In this case, since you just want to check 2 specific keys, it might be better to do:

```go
So(body["service"].(string), ShouldEqual, "test-services")
So(int(body["status"].(float64)), ShouldEqual, http.StatusOK)
```

Reply:
I believe you're inferring that the for loop may be more efficient since you're not searching through a map on each `body[string]` statement, but there's not too many `body[string]` so the efficiencies are negligible.

Reply:
My point is, the loop is ok as long as all the iterations are the same. In this case you need to check the key name and do something different base on it. It means that, if you need to check for another key, you would have to add another `if` and so on. And at that point the loop loses its benefits.